jpa/subsystem/src/main/java/org/jboss/as/jpa/config/PersistenceUnitMetadataImpl.java

```diff
@@ -379,6 +379,9 @@ public boolean needsJPADelegatingClassFileTransformer() {
 
     @Override
     public void addTransformer(ClassTransformer classTransformer) {
+        if(classTransformer.getClass().getName().contains("org.hibernate")) {
```

Comment:
This should be `startsWith` rather than `contains`. My custom `com.acme.integration.org.hibernate.transformers` would be inadvertently skipped. :-)